modules/ROOT/pages/queries/composed-queries/sequential-queries.adoc

```diff
@@ -0,0 +1,302 @@
+====
+
+
+== Interactions with `CALL` subqueries
+
+You can use `NEXT` to rewrite queries containing  `CALL` subqueries.
+
+[[call-subquery-rewrite]]
+.Rewriting a query with a `CALL` subquery
+====
+[cols="1,1"]
+|===
+a|
+.`CALL` subquery
+[source, cypher]
+----
+MATCH (p:Product)
+CALL (p) {
+    MATCH (c:Customer)-[:BUYS]->(p)
+    RETURN collect(c.firstName) AS customers
+}
+RETURN p.name as product, customers
+----
+a|
+.`NEXT`
+[source, cypher]
+----
+MATCH (p:Product)
+RETURN p
+
+NEXT
+
+MATCH (c:Customer)-[:BUYS]->(p)
+RETURN collect(c.firstName) AS customers, p
+
+NEXT
+
+RETURN p.name as product, customers
+----
+|===
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| product      | customers
+
+| "Laptop"     | ["Amir", "Mateo", "Leila", "Yusuf"]
+| "Phone"      | ["Niko"]
+| "Headphones" | ["Keisha", "Hannah", "Niko"]
+| "Chocolate"  | ["Amir", "Mateo", "Yusuf"]
+| "Coffee"     | ["Mateo", "Hannah", "Niko"]
+2+d|Rows: 5
+|===
+====
+
+Even though the `NEXT` query in <<call-subquery-rewrite>> has more lines, it is divided into three segments which are easy to read.
+It also avoids the parentheses and indentation of the `CALL` subquery.
+
+Variables which are local to a query and which are not explicitly returned are not accessible by subsequent queries in the context of `NEXT`. This allows you to control variable scope similarly to what you can do with `WITH`, see xref:clauses/with.adoc#variable-scope[Control variables in scope].
+[NOTE]
+====
+`NEXT` cannot be used inside a `CALL` subquery that uses the (deprecated) xref:subqueries/call-subquery.adoc#importing-with[importing `WITH`] syntax.
+====
+
+
+== Interactions with conditional queries
+
+[[next-and-conditional]]
+. Conditional queries in `NEXT`
+====
+.Query
+[source, cypher]
+----
+MATCH (c:Customer)-[:BUYS]->(:Product)<-[:SUPPLIES]-(s:Supplier)
+RETURN c.firstName AS customer, s.name AS supplier
+
+NEXT
+
+WHEN supplier = "TechCorp" THEN
+  RETURN customer, "Tech enjoyer" AS personality
+WHEN supplier = "Foodies Inc." THEN
+  RETURN customer, "Tropical plant enjoyer" AS personality
+  
+NEXT
+
+RETURN customer, collect(DISTINCT personality) AS personalities
+
+NEXT
+
+WHEN size(personalities) > 1 THEN
+  RETURN customer, "Enjoyer of tech and plants" AS personality
+ELSE
+  RETURN customer, personalities[0] AS personality
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| customer | personality
+
+| "Amir"   | "Enjoyer of tech and plants"
+| "Mateo"  | "Enjoyer of tech and plants"
+| "Yusuf"  | "Enjoyer of tech and plants"
+| "Niko"   | "Enjoyer of tech and plants"
+| "Hannah" | "Enjoyer of tech and plants"
+| "Leila"  | "Tech enjoyer"
+| "Keisha" | "Tech enjoyer"
+
+2+d|Rows: 7
+|===
+====
+
+In <<next-and-conditional>>, customers are assigned personality types based on the products they purchased.
```

Comment:
this does not render - also, I am not sure about linking to examples just above. Couldn't we just say "In the example above.."?
```suggestion
In the above query, customers are assigned personality types based on the products they purchased.
```